test/database/databaseTest.cpp

```diff
@@ -0,0 +1,17 @@
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+#include <database/database.cpp>
+
+using namespace ::testing;
+
+TEST(DatabaseLogingTest, ExpectedValidLoginToWork)
```

Comment:
Consider adding more tests.

Reply:
I test everything in this one. I did every EXPECT_FALSE/TRUE in separate tests, but @ddnowicki told me not to spam with tests so I moved everything into one test. Should I change it back to separated tests?

Reply:
Imo, currently there is no point in adding more tests to the same one, because it will be difficult to find out later when there are 50 tests and one fails.